src/components/WeatherComponent.tsx

```diff
@@ -20,22 +19,13 @@ const WeatherComponent = () => {
   }, []);
 
   return (
-    <Container>
-      <Title order={2}>
-        {weather?.forcast === 'Sunny' ? (
-          <MdWbSunny data-testid="sunny" />
-        ) : weather?.forcast === 'Snowing' ? (
-          <BsFillCloudSnowFill data-testid="snowing" />
-        ) : (
-          <IoIosPartlySunny data-testid="overcast" />
-        )}
-      </Title>
-
-      <Text size="xl" data-testid="forcast">
-        {weather?.forcast}
-      </Text>
-      <Text size="lg" data-testid="temp">
-        Temp: {`${weather?.min} to ${weather?.max}`}
+    <Container style={{ width: '100%' }}>
+      <WeatherContext.Provider value={weather?.forcast}>
+        <UserForm />
```

Comment:
Good to see the use of a Context provider. I would take the UserForm and the weather context out of this component. We can leave this here for now, I'll create an issue related to this and we can refactor it later